src/main/java/com/example/AllPermutationPrime.java

```diff
@@ -0,0 +1,62 @@
+package com.example;
+
+import java.util.ArrayList;
+
+/**
+ * Készíts egy olyan függvényt, aminek
+ * bemenő paramétere n,
+ * és válaszként kilistázza azokat a számokat 1-től n-ig,
+ * amelyeknek az összes körkörös permutációja prím.
+ */
+public class AllPermutationPrime {
+
+    private static boolean isPrime(int n) {        
+        boolean prime = true;
+        int divider = n - 1;
```

Comment:
There are hundreds of pages about prime testing so I don't think we will change the world here but I m curious why did you choose this approach? 
If you start bottom-up (starting at `divider=2`) for every second number it will be quickly checked it's not a prime. With your approach you potentially climb down to half of the number or further, doing lot more division than otherwise. Am I right?

Reply:
Absolutely. OK.